hack/build_rpms.sh

```diff
@@ -4,11 +4,10 @@ if [ -f /etc/debian_version ]; then
     echo "Building RPMs on a non-rpm based system via podman..."
     cat >bin/build_rpms.sh <<EOF
 #!/usr/bin/env bash
-dnf install -y make packit golang openssl-devel
 cd /work
 ./hack/build_rpms_packit.sh
 EOF
-    podman run --rm -t -v $(pwd):/work fedora:39 bash /work/bin/build_rpms.sh
+    podman run --rm -t -v $(pwd):/work quay.io/flightctl/ci-rpm-builder:latest bash /work/bin/build_rpms.sh
```

Comment:
This saves 2+ minutes of build time, I need to upload the containerfile for this and setup CI.